src/OnlySpans.PolyLeads.Api/Exceptions/RoleManagementException.cs

```diff
@@ -1,7 +1,18 @@
-﻿namespace OnlySpans.PolyLeads.Api.Exceptions;
+﻿using System.Diagnostics;
+
+namespace OnlySpans.PolyLeads.Api.Exceptions;
 
 public sealed class RoleManagementException : ApiException
 {
     public RoleManagementException(string? message, Exception? innerException = null)
         : base(message, innerException) { }
+
+    [StackTraceHidden]
+    public static void ThrowIfNotExist(string roleName)
+    {
+        var allRoles = new[] {"Student", "StudentUnionOrganizer", "Headman", "Admin"};
```

Comment:
тут не должно быть такой логики, в хэндлере надо делать

Reply:
вообще давай заведем класс ApplicationRoleName и туда костантами роли пометстим

```suggestion
        pubic static class ApplicationRoleName
        {
            public static IReadOnlyList<string> All { get; } = [Student, ...];
        
            public const string Student = "Student";
            
            ...
        }
```

во всех местах соответсвенно заменить надо будет